code/cli/src/operational.rs

```diff
@@ -0,0 +1,634 @@
+const INVESTORS_VESTING_END_DAY: u32 = 01;
+
+const INFRA_PROVIDERS_VESTING_START_YEAR: u32 = 2023;
+const INFRA_PROVIDERS_VESTING_START_MONTH: u32 = 04;
+const INFRA_PROVIDERS_VESTING_START_DAY: u32 = 01;
+
+const INFRA_PROVIDERS_VESTING_END_YEAR: u32 = 2023;
+const INFRA_PROVIDERS_VESTING_END_MONTH: u32 = 07;
+const INFRA_PROVIDERS_VESTING_END_DAY: u32 = 01;
+
+const VESTING_CLAIM_STEP_DAYS: u32 = 1;
+
+const ORACLE_FUNDS: Amount<Raw> = Amount::new(72_132_699);
+
+const fn _infrastructure_total_funds() -> Amount<Raw> {
+	let mut amount = Amount::new(0);
+	let mut i = 0;
+	while i < INFRASTRUCTURE_PROVIDERS.len() {
+		let (_, reward) = INFRASTRUCTURE_PROVIDERS[i];
+		amount = amount + reward;
+		i += 1;
+	}
+	amount
+}
+
+// The value was rounded down, 700 -> 699
+const _INFRASTRUCTURE_FUNDS_TOTAL_EXPECTED: Amount<Raw> = Amount::new(72_132_699);
+const _INFRASTRUCTURE_FUNDS_ASSERT: () =
+	assert!(_infrastructure_total_funds() == _INFRASTRUCTURE_FUNDS_TOTAL_EXPECTED);
+
+const fn _investor_total_funds() -> Amount<Raw> {
+	let mut amount = Amount::new(0);
+	let mut i = 0;
+	while i < INVESTORS.len() {
+		let (_, reward) = INVESTORS[i];
+		amount = amount + reward;
+		i += 1;
+	}
+	i = 0;
+	while i < MISSING_INVESTORS.len() {
+		amount = amount + MISSING_INVESTORS[i];
+		i += 1;
+	}
+	amount
+}
+
+const _OPERATIONAL_FUNDS_TOTAL: Amount<Raw> = _investor_total_funds() +
+	TREASURY_MSIG_FUNDS +
+	ECOSYSTEM_INCENTIVES_MSIG_FUNDS +
+	LIQUIDITY_PROGRAMS_MSIG_FUNDS +
+	TEAM_MSIG_FUNDS;
+
+// We don't compute the crowdloan here.
+const _OPERATIONAL_FUNDS_TOTAL_EXPECTED: Amount<Raw> = Amount::new(7_000_000_000);
+const _OPERATIONAL_FUNDS_ASSERT: () =
+	assert!(_OPERATIONAL_FUNDS_TOTAL == _OPERATIONAL_FUNDS_TOTAL_EXPECTED);
+
+#[derive(Debug)]
+pub enum OperationalError {
+	Common(CommonError),
+	ApiFailure(substrate_api_client::ApiClientError),
+}
+
+impl From<substrate_api_client::ApiClientError> for OperationalError {
+	fn from(x: substrate_api_client::ApiClientError) -> Self {
+		Self::ApiFailure(x)
+	}
+}
+
+impl From<CommonError> for OperationalError {
+	fn from(x: CommonError) -> Self {
+		Self::Common(x)
+	}
+}
+
+pub fn setup_native_council(
+	api: Api<sp_core::sr25519::Pair, WsRpcClient, AssetTipExtrinsicParams>,
+) -> Result<(), OperationalError> {
+	let native_council_members = NATIVE_COUNCIL
+		.iter()
+		.map(|x| extract_account(x))
+		.collect::<Result<Vec<_>, _>>()?;
+
+	log::info!("Submitting native council...");
+	let tx_hash = api_wrap::<_, OperationalError>(
+		api.send_extrinsic(
+			compose_extrinsic!(
+				api,
+				"Sudo",
+				"sudo",
+				compose_call!(
+					api.metadata.clone(),
+					"Council",
+					"set_members",
+					// NewMembers
+					native_council_members,
+					// Prime
+					None::<AccountId32>,
+					// OldCount
+					0u32
+				)
+			)
+			.hex_encode(),
+			XtStatus::InBlock,
+		),
+	)?;
+	log::info!("Native council submitted, hash={:?}", tx_hash);
+
+	Ok(())
+}
+
+pub fn setup_technical_council(
```

Comment:
Where do we set threshold for the technical council to 4 ?

Reply:
fixed